tests/test_reconciliation.py

```
@@ -200,7 +199,7 @@ def test_not_local_does_not_fire(self):
             cvss_vector="CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:H",
         )
         SuggestImpact.reconcile_severity(out, "t", _clf(impact="MODERATE"))
-        assert out.impact != "LOW" or "H8" not in ""
+        assert out.impact != "LOW" or "H8" not in ""  # noqa: PLR0133
```

Ruff (0.16.0)

[warning] Unnecessary membership test on empty collection

(RUF060)